test/local/routes/oauth.js

```diff
@@ -87,31 +92,42 @@ describe('/oauth/ routes', () => {
       });
       assert.deepEqual(resp, { key: 'data' });
     });
+  });
 
-    it('rejects an `assertion` parameter in the request payload', async () => {
```

Comment:
This test wasn't actually doing what it said on the tin, so I removed it.  There's an equivalent test added in the remote tests, where it'll work properly.